x/slashing/keeper.go

```diff
@@ -183,7 +183,16 @@ func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr crypto.Address, p
 		// Array value at this index has not changed, no need to update counter
 	}
 
+	// Emit warning event if Validator misses block
 	if missed {
+		ctx.EventManager().EmitEvent(
+			sdk.NewEvent(
+				types.EventTypeSlash,
```

Comment:
```suggestion
				types.EventTypeLiveness,
```

We're not slashing anyone here 🐸

Reply:
🔪